### ISSUE TITLE
OSX port / maintener

### ISSUE THREAD

**Issue by ocornut** [ocornut/meka#7]
Following discussion on 
http://www.smspower.org/forums/15120-MEKANeedsAMacOSXDeveloperMaintainer
cc: @blind

blind's fork is
https://github.com/blind/meka/tree/feature-macosx

Sorry I didn't have time to look at it. I'd really like to move that forward.
The reason I included binaries for Windows is that it is such a nightmare to compile anything under Windows. However my intuition was that OSX had good-enough packaging system that you could grab Allegro from somewhere? Just curious.

**Comment by blind** [comment]
It was a few month ago that I set this up, so I can't remember if I tried to use allegro from some packaging system at first. But what I do remember is that I had to grab a patch from a more recent version of Allegro to keep it from crashing if a controller was plugged in. So the libs I added to the repo are the patched ones I built myself. I wanted to use the same Allegro version as the Windows build used.

It seems that Allegro 5.0.11 is available through brew, so I can try to change the Makefile to use the libs provided there instead. With the possibility that MEKA crashes with a controller.

**Comment by ocornut** [comment]
Might be a new package perhaps. Bit annoying it has that sort of bugs.
If you end up with using the package would be nice to squash or redo the branch so that it doesn't carry the file size for the unused allegro binaries. It completely forgot that part when moving from SVN to Git but the Windows binaries are probably not a great idea there.

**Comment by blind** [comment]
I just tried it and the crash seems to be fixed in 5.0.11.

I'll remove the current feature-macosx branch and replace it with one where the allegro libs are not committed. I just hope nobody merged it yet.

**Comment by dajero** [comment]
I have been trying to get the code above running on my machine (running Yosemite), and the binary compiles just fine. I also managed to get an application bundle built by altering some lines in the dist_bin_osx.sh script.

Is work still being done on the OS X port? Should I file a pull request to @blind 's repository?

**Comment by ocornut** [comment]
I'll do it now. Sorry I didn't realise the branch had been updated without the binaries.
Thanks a lot @blind !

**Comment by ocornut** [comment]
OK i am back here - @blind afaik your branch doesn't have any remaining changes right now?

**Comment by blind** [comment]
That is correct, I haven't done any changes since @dajero s osx support commit.

**Comment by gingerbeardman** [comment]
Where can I find the latest binary?

Or (easily) buildable Mac source?

**Comment by ocornut** [comment]
As pointed by this thread there's no Mac maintainer and probably no trivially buildable sources.

**Comment by gingerbeardman** [comment]
OK, thanks.

I'm running the latest Windows beta/proto using CrossOver/Wine on my Mac. Works well and got the debug information I needed!
